packages/axios-extension/src/abap/lrep-service.ts

```diff
@@ -76,6 +76,20 @@ export interface Message {
     variables?: string[];
 }
 
+/**
+ * Structure of the system info reponse data.
+ */
+export interface SystemInfo {
+    adaptationProjectTypes: string[];
```

Comment:
Can you define the `adaptationProjectTypes` values?

Reply:
Fixed

Reply:
Closing for @longieirl since he is ooo